src/Microsoft.AspNetCore.Sockets.Client.Http/Internal/ServerSentEventsTransport.cs

```diff
@@ -131,6 +157,13 @@ public Task StartAsync(Uri url, IDuplexPipe application, TransferFormat transfer
                                     }
                                     break;
                             }
+
+                            // We canceled in the middle of applying back pressure
+                            // or if the consumer is done
+                            if (flushResult.IsCanceled || flushResult.IsCompleted)
+                            {
+                                break;
```

Comment:
Add `Log.EventStreamEnded(_logger);` ?

Reply:
No, the application stopped reading 😄 . I don't think we need a log for this.

Reply:
> I don't think we need a log for this.

Famous last words :trollface: